src/util/container.h

```diff
@@ -196,16 +196,17 @@ class MutexedQueue
 
 	MutexedQueue()
 	{
+		std::deque<T>();
 	}
 	bool empty()
 	{
 		JMutexAutoLock lock(m_mutex);
-		return (m_size.GetValue() == 0);
+		return (std::deque<T>::size() == 0);
```

Comment:
No:
/home/nerzhul/Devel/minetest/src/util/container.h:204:16: erreur: there are no arguments to ‘size’ that depend on a template parameter, so a declaration of ‘size’ must be available [-fpermissive]
   return (size() == 0);